drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -805,6 +803,11 @@ static int i2c_it8xxx2_transfer(const struct device *dev, struct i2c_msg *msgs,
 		/* Wait for the transfer to complete */
 		/* TODO: the timeout should be adjustable */
 		res = k_sem_take(&data->device_sync_sem, K_MSEC(100));
+		/*
+		 * Ensure that interrupt can be disabled immediately when
+		 * the timeout occurs.
+		 */
+		irq_disable(config->i2c_irq_base);
```

Comment:
When does this get enabled/re-enabled?

Reply:
Yes the commit messages needs a motivation. It should always be the first thing in the commit message

Reply:
The commit message and  comment are modified.